src/util/customTransformerFactory.js

```diff
@@ -20,6 +20,7 @@ const UserTransformHandlerFactory = (userTransformation) => {
     runUserTransfrom: async (events, testMode, libraryVersionIDs) => {
       switch (userTransformation.language) {
         case 'pythonfaas':
+          userTransformation.libraryVersionIds = libraryVersionIDs;
           return runOpenFaasUserTransform(events, userTransformation, testMode);
```

Comment:
```suggestion
          return runOpenFaasUserTransform(events, userTransformation, libraryVersionIDs, testMode);
```
can pass param directly in function. instead of enriching userTransformation